### PR TITLE
use "comment" over "comments" in review buffer

### LINKED ISSUE #648: Use comment over comments in review buffer
1 comments shows up in the review buffer which is grammatically incorrect.

### PR DESCRIPTION
**Pull request by macovsky**
### Describe what this PR does / why we need it

Displays "comment" in review buffer instead of "comments"

### Does this pull request fix one issue?

Fixes #648

### Describe how to verify it

Make a comment in your review and see that it's displayed as "1 comment" in diff

### Checklist

- [ ] Passing tests and linting standards
- [ ] Documentation updates in README.md and doc/octo.txt

### CODE REVIEW

File: lua/octo/reviews/file-entry.lua [hunk 1]
```diff
@@ -382,7 +382,9 @@ function FileEntry:place_signs()
 
           -- place the virtual text only on first line
           local last_date = comment.lastEditedAt ~= vim.NIL and comment.lastEditedAt or comment.createdAt
-          local vt_msg = string.format("    %d comments (%s)", #thread.comments.nodes, utils.format_date(last_date))
+          local comments_count = #thread.comments.nodes
+          local comments_word = comments_count == 1 and "comment" or "comments"
```
Comment (wd60622) [inline lua/octo/reviews/file-entry.lua:386]: Test multiline comment with branch checked out

File: lua/octo/reviews/file-entry.lua [hunk 2]
```diff
@@ -382,7 +382,9 @@ function FileEntry:place_signs()
 
           -- place the virtual text only on first line
           local last_date = comment.lastEditedAt ~= vim.NIL and comment.lastEditedAt or comment.createdAt
-          local vt_msg = string.format("    %d comments (%s)", #thread.comments.nodes, utils.format_date(last_date))
```
Comment (wd60622) [inline lua/octo/reviews/file-entry.lua:385]: Test comment over here

**Review by wd60622 — COMMENTED** [review]
I think this seems good. I checked out the branch and had the \"comments\" still when I added the comment but \"comment\" after loaded. I tried again and it wasn't like that.

Do you know if there is a different function at comment creation and rendering?

**Comment by macovsky** [comment]
> I think this seems good. I checked out the branch and had the \"comments\" still when I added the comment but \"comment\" after loaded. I tried again and it wasn't like that.

hm, I tried multiple times and couldn't see "comments" anywhere, it just appears as "comment" for me right from the start. did you see it as "1 comments" or maybe even "0 comments"?

> Do you know if there is a different function at comment creation and rendering?

idk really, just looked in the code and couldn't find anything related

**Review by wd60622 — APPROVED** [review]
Makes sense. Thanks for taking a look. Lets push it through!

**Pull request merged**